docs/csharp/tour-of-csharp/classes-and-objects.md

```diff
@@ -63,9 +63,9 @@ Each member of a class has an associated accessibility, which controls the regio
 * `protected`
 	- Access limited to this class or classes derived from this class
```

Comment:
Since `protected` is applied to a member, it's better to phrase this as, "Access limited to the containing class or classes derived from the containing class."